### PR TITLE
WIP- P8s Getting started - beta

### PR DESCRIPTION
**Pull request by YajB**
# What changed

New content: https://deploy-preview-728--logz-docs.netlify.app/user-guide/infrastructure-monitoring/p8s-getting-started
and child topics. 

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - metrics integrations
+contributors:
+  - yberlinger
+---
+This topic is for Prometheus users migrating their solution to Logz.io Infrastructure Monitoring powered by Grafana.
+
+You can send your Prometheus metrics to your Logz.io Infrastructure Monitoring account.
+To do this, you’ll need to send your application metrics to Logz.io in JSON format and stick to the formatting guidelines.
+
+Our log scraping software service takes all your log file data, scrapes it and stores it in an archive. The log data may include messaging, calendaring, and other data types.
```
Comment (Noa-Levi) [inline _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:17]: what is the log scrapping related here?

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  logzio-plan: community
+tags:
+  - metrics integrations
+contributors:
+  - yberlinger
+---
+This topic is for Prometheus users migrating their solution to Logz.io Infrastructure Monitoring powered by Grafana.
+
+You can send your Prometheus metrics to your Logz.io Infrastructure Monitoring account.
+To do this, you’ll need to send your application metrics to Logz.io in JSON format and stick to the formatting guidelines.
```
Comment (Noa-Levi) [inline _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:15]: I don't think we should put this here, maybe confusing (the JSON format note)
the user just defines the remote write, we are in charge od the format that gets to the listener

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Our log scraping software service takes all your log file data, scrapes it and stores it in an archive. The log data may include messaging, calendaring, and other data types. 
+
+Before you begin, make sure you have the following details for your Logz.io Metrics account: Listener address (URL) and Logz.io metrics account token.
+
```
Comment (Noa-Levi) [inline _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:20]: maybe also add explanations or links where he can find this data

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+Notes:
```
Comment (Noa-Levi) [inline _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:21]: I think it is confusing to put this at this part, should be after the remote write process is done

Comment (YajB) [reply _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:21]: But don't you need to put the token and listener details *in* your Prometheus?

File: _source/no-search/p8s-remote-write.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+1. Define Logz.io as a new endpoint, using remote write.
+2. Provide the Logz.io metrics account token and listener address (URL).
+
+  - To find the default token in the [General Settings](https://app.logz.io/#/dashboard/settings/general) page , click **<i class="li li-gear"></i> > Settings > General** in the top menu.
+
+<video autoplay loop>
+    <source src="https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana-videos/p8sgo-to-acct_settings2.mp4" type="video/mp4"/>
+</video>
+
+  - To find the correct listener URL for your region, look [here](/user-guide/log-shipping/listener-ip-addresses.html).
```
Comment (Noa-Levi) [inline _source/no-search/p8s-remote-write.md:51]: can we make this link and the one above to be opened on a separate page? so the user will not leave the current instructions

File: _source/no-search/p8s-remote-write.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+Integrate your Prometheus toolkit with the Logz.io Metrics platform to continuously monitor your system’s health and help you maintain system performance, speed, and resilience, and address problematic trends when they first appear - and before they become critical and threaten crashes or system outages.
+
+![Showcased Metrics dashboard](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/metrics-intro.png)
+
+To send your Prometheus application metrics to a Logz.io Infrastructure Monitoring account, use remote write to connect to Logz.io as the endpoint. Your data is formatted as JSON documents by the Logz.io listener. 
+
+If you have multiple Prometheus server instances, you'll have to add Logz.io as an endpoint for each instance. 
+
+By default, all metrics in your Prometheus server(s) will be sent to Logz.io. To drop or send specific metrics, add Prometheus labeling before enabling the remote write, or as part of the remote write configuration. You can read more about Prometheus relabeling tricks <a href ="https://medium.com/quiq-blog/prometheus-relabeling-tricks-6ae62c56cbda" target="_blank">here. <i class="fas fa-external-link-alt"></i> </a>
```
Comment (Noa-Levi) [inline _source/no-search/p8s-remote-write.md:23]: is it ok to use this guy blog as a reliable source from our docs?

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md (hunk 1 above)
Comment (YajB) [reply _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:17]: removed it - error in understanding

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md (hunk 3 above)
Comment (YajB) [reply _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:20]: Done

File: _source/user-guide/infrastructure-monitoring/p8s-remote-write.md (hunk 4 above)
Comment (YajB) [reply _source/user-guide/infrastructure-monitoring/p8s-remote-write.md:21]: Reorganized

File: _source/no-search/p8s-remote-write.md (hunk 6 above)
Comment (YajB) [reply _source/no-search/p8s-remote-write.md:23]: These are the links that Doron shared. @Doron-Bargo  - what do you say?

File: _source/no-search/p8s-remote-write.md (hunk 5 above)
Comment (YajB) [reply _source/no-search/p8s-remote-write.md:51]: Absolutely. Done.

**Comment by YajB** [comment]
content handled in different PR

**Pull request closed without merging**